src/FStringMatchRule.js

```diff
@@ -0,0 +1,167 @@
+/*
+ * FStringMatchRule.js - implement a rule to match f-string substitution parameters
+ *
+ * Copyright © 2022 JEDLSoft
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Locale from 'ilib-locale';
+import { Rule, Result } from 'i18nlint-common';
+
+// from https://peps.python.org/pep-0498/
+const fstringRegExp = /\{(\}\}|[^}])*?\}/g;
+
+/**
+ * @class Represent an i18nlint rule.
+ */
+class FStringMatchRule extends Rule {
+    constructor(options) {
+        super(options);
+        this.name = "resource-python-fstrings-match";
```

Comment:
regarding the rule name, are there specific rules? or anything acceptable?
and How about type? Is not necessary this info? i.e )resource-matcher, resource-source a